sdk/core/azure-core/src/main/java/com/azure/core/implementation/http/rest/RestProxyBase.java

```diff
@@ -118,7 +118,9 @@ public final Object invoke(Object proxy, final Method method, RequestOptions opt
         }
     }
 
-    protected abstract Object invoke(Object proxy, Method method, RequestOptions options, EnumSet<ErrorOptions> errorOptions, Consumer<HttpRequest> httpRequestConsumer, SwaggerMethodParser methodParser, HttpRequest request, Context context);
+    protected abstract Object invoke(Object proxy, Method method, RequestOptions options,
```

Comment:
Added to trigger Core CI but we'll want this code style change anyways, so leaving it